ui.tests/test-module/specs/datepicker/datepicker.runtime.spec.js

```diff
@@ -221,4 +224,47 @@ describe("Form Runtime with Date Picker", () => {
       });
     });
 
+    it("input given as per edit pattern should be valid", () => {
+        const [datePicker8] = Object.entries(formContainer._fields)[7]
+        const incorrectInput = "01-01-2023";
+        const correctInput = "25/2/2023";
+        cy.get(`#${datePicker8}`).find("input").clear().type(incorrectInput).blur().then(x => {
+            cy.get(`#${datePicker8}`).find(".cmp-adaptiveform-datepicker__errormessage").should('have.text',"Specify the value in allowed format : date.")
+        });
+        cy.get(`#${datePicker8}`).find("input").clear().type(correctInput).blur().then(x => {
+            cy.get(`#${datePicker8}`).find(".cmp-adaptiveform-datepicker__errormessage").should('have.text',"");
+        });
+    });
+
+    it("Display pattern transforms to edit format when focused", () => {
+        const [datePicker8, fieldView] = Object.entries(formContainer._fields)[7]
+        const input = "25/2/2023", displayFormat = "February 25, 2023";
+        cy.get(`#${datePicker8}`).should('have.class', 'cmp-adaptiveform-datepicker--empty');
+        cy.get(`#${datePicker8}`).find("input").clear().type(input).blur().then(() => {
+            cy.get(`#${datePicker8}`).find(".cmp-adaptiveform-datepicker__errormessage").should('have.text',"");
+            cy.get(`#${datePicker8}`).find("input").should('have.value', displayFormat);
+            cy.get(`#${datePicker8}`).find("input").click().then(() => {
+                cy.get(`#${datePicker8}`).find("input").should('have.value',input);
+                })
+        });
+    });
+
+    it("Test changing dates in datePicker then typing manually updates model", () => {
+        const [datePicker7] = Object.entries(formContainer._fields)[6];
+
+        cy.get(`#${datePicker7}`).find(".cmp-adaptiveform-datepicker__calendar-icon").should("be.visible")
+            .eq(0).click({force: true}).then(() => {
```

Comment:
No use of force:true on runtime test

Reply:
force true is used because the calendar icons are overlapping each other because the test collaterals do not have css